frontend/src/components/Footer.astro

```diff
@@ -0,0 +1,4 @@
+<div class="flex w-full justify-center items-center h-10">
+  <!-- <a href="/privacy-and-happy-lawyers">Terms & Privacy</a> -->
+  <a href="/privacy">Terms & Privacy</a>
```

Comment:
This seems to be the wrong url

Reply:
Astro generates routes based on file names. To get them in sync on the frontend when developing (`astro dev`) and when serving from the server, it would be easiest to have the names in sync.

Should we change the `privacy.astro` into `privacy-and-happy-lawyers.astro` then?

Reply:
Yea, gotta have happy lawyers haha

Reply:
Done in https://github.com/oeb25/Cuely/commit/ce070d2f82005367633f24de5089ebbd5ed3f887